.github/workflows/main.yml

```diff
@@ -4,6 +4,12 @@ on: ['push', 'pull_request']
 env:
   BUILD_TYPE: Release
   VCPKG_DEFAULT_TRIPLET: x64-windows
+  VCPKG_DEFAULT_BINARY_CACHE: ${{ github.workspace }}\vcpkg-cache
```

Comment:
%LOCALAPPDATA%\vcpkg\archives is the default binary cache for vcpkg under windows, is there a reason you are not using it?

Reply:
I wanted to have all resources to be inside of workspace directory so you wouldn't have to rely on defaults (just in case those defaults ever change). As you point out, it was not necessary.